test/fixtures/controller.bar/floatBar/float-bar-horizontal.json

```diff
@@ -0,0 +1,43 @@
+{
+    "config": {
+        "type": "horizontalBar",
+        "data": {
+            "labels": ["2030", "2034", "2038", "2042"],
+            "datasets": [{
+                "backgroundColor": "#FF6384",
+                "data": [11, [6,2], [-4,-7], -2]
+            }, {
+                "backgroundColor": "#36A2EB",
+                "data": [[1,2], [3,4], [-2,-3], [1,4]]
+            }, {
+                "backgroundColor": "#FFCE56",
+                "data": [[0,1], [1,2], [-2,-1], [1,-7]]
+            }]
+        },
+        "options": {
+            "title": false,
+            "legend": false,
+            "scales": {
+                "xAxes": [{
+                    "ticks": {
+                        "source": "labels",
+                        "display": false
+                    }
+                }],
+                "yAxes": [{
+                    "ticks": {
```

Comment:
I think we should not display x and y axes at all to minimize the scope of the test. And, `ticks.min` and `ticks.max` should be specified to make sure that the correct range is displayed. The same applies to the other tests.
```json
"scales": {
    "xAxes": [{
        "display": false,
        "ticks": {
            "min": -8,
            "max": 12
        }
	}],
    "yAxes": [{
        "display": false
    }]
}
```

Reply:
totally, agree. Changing